src/robusta/runner/config_loader.py

```diff
@@ -177,6 +187,22 @@ def __reload_playbook_packages(self, change_name):
                 runner_config.playbook_repos[
                     "robusta.core.playbooks.internal"
                 ] = PlaybookRepo(url=INTERNAL_PLAYBOOKS_ROOT, pip_install=False)
+                # order matters! Loading the default first, allows overriding it if adding package with the same name
+                # default playbooks
+                runner_config.playbook_repos[
+                    self.__get_package_name(DEFAULT_PLAYBOOKS_ROOT)
+                ] = PlaybookRepo(url=f"file://{DEFAULT_PLAYBOOKS_ROOT}")
+
+                # custom playbooks
+                if os.path.exists(CUSTOM_PLAYBOOKS_ROOT):
+                    for custom_playbooks_location in os.listdir(CUSTOM_PLAYBOOKS_ROOT):
+                        location = os.path.join(CUSTOM_PLAYBOOKS_ROOT, custom_playbooks_location)
+                        runner_config.playbook_repos[
```

Comment:
One bad package will prevent all playbooks from loading. (Because of the exception thrown in `self.__get_package_name`. Should we change that?

Reply:
I think it's fine. 
You pushed a bad playbooks package, the whole configuration is rejected.
If you think it's important, I can change the behavior. should I?

Reply:
No, it's not important.